pom.xml

```diff
@@ -306,18 +306,10 @@
 			<scope>test</scope>
 		</dependency>
 		<dependency>
-			<groupId>uk.org.lidalia</groupId>
-			<artifactId>slf4j-test</artifactId>
-			<version>1.1.0</version>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-simple</artifactId>
+			<version>[1.7,2)</version>
```

Comment:
The version number looks strange. Does it mean "1.7.2"? Why don't we use the latest version, such as "1.7.25"?

Reply:
It's an interval, it's because it's a library and the projects using our library might want to use another version